seq/alphabet_conv.go

```diff
@@ -0,0 +1,638 @@
+var rnaCodonToAminoAcidTranslTable29 = map[string]string{
+	"UUU": "F", "UCU": "S", "UAU": "Y", "UGU": "C",
+	"UUC": "F", "UCC": "S", "UAC": "Y", "UGC": "C",
+	"UUA": "L", "UCA": "S", "UAA": "Y", "UGA": "*",
+	"UUG": "L", "UCG": "S", "UAG": "Y", "UGG": "W",
+	"CUU": "L", "CCU": "P", "CAU": "H", "CGU": "R",
+	"CUC": "L", "CCC": "P", "CAC": "H", "CGC": "R",
+	"CUA": "L", "CCA": "P", "CAA": "Q", "CGA": "R",
+	"CUG": "L", "CCG": "P", "CAG": "Q", "CGG": "R",
+	"AUU": "I", "ACU": "T", "AAU": "N", "AGU": "S",
+	"AUC": "I", "ACC": "T", "AAC": "N", "AGC": "S",
+	"AUA": "I", "ACA": "T", "AAA": "K", "AGA": "R",
+	"AUG": "M", "ACG": "T", "AAG": "K", "AGG": "R",
+	"GUU": "V", "GCU": "A", "GAU": "D", "GGU": "G",
+	"GUC": "V", "GCC": "A", "GAC": "D", "GGC": "G",
+	"GUA": "V", "GCA": "A", "GAA": "E", "GGA": "G",
+	"GUG": "V", "GCG": "A", "GAG": "E", "GGG": "G",
+}
+
+//https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=tgencodes#SG30
+var rnaCodonToAminoAcidTranslTable30 = map[string]string{
+	"UUU": "F", "UCU": "S", "UAU": "Y", "UGU": "C",
+	"UUC": "F", "UCC": "S", "UAC": "Y", "UGC": "C",
+	"UUA": "L", "UCA": "S", "UAA": "G", "UGA": "*",
+	"UUG": "L", "UCG": "S", "UAG": "G", "UGG": "W",
+	"CUU": "L", "CCU": "P", "CAU": "H", "CGU": "R",
+	"CUC": "L", "CCC": "P", "CAC": "H", "CGC": "R",
+	"CUA": "L", "CCA": "P", "CAA": "Q", "CGA": "R",
+	"CUG": "L", "CCG": "P", "CAG": "Q", "CGG": "R",
+	"AUU": "I", "ACU": "T", "AAU": "N", "AGU": "S",
+	"AUC": "I", "ACC": "T", "AAC": "N", "AGC": "S",
+	"AUA": "I", "ACA": "T", "AAA": "K", "AGA": "R",
+	"AUG": "M", "ACG": "T", "AAG": "K", "AGG": "R",
+	"GUU": "V", "GCU": "A", "GAU": "D", "GGU": "G",
+	"GUC": "V", "GCC": "A", "GAC": "D", "GGC": "G",
+	"GUA": "V", "GCA": "A", "GAA": "E", "GGA": "G",
+	"GUG": "V", "GCG": "A", "GAG": "E", "GGG": "G",
+}
+
+//https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=tgencodes#SG31
+var rnaCodonToAminoAcidTranslTable31 = map[string]string{
+	"UUU": "F", "UCU": "S", "UAU": "Y", "UGU": "C",
+	"UUC": "F", "UCC": "S", "UAC": "Y", "UGC": "C",
+	"UUA": "L", "UCA": "S", "UAA": "G", "UGA": "Y",
+	"UUG": "L", "UCG": "S", "UAG": "G", "UGG": "W",
+	"CUU": "L", "CCU": "P", "CAU": "H", "CGU": "R",
+	"CUC": "L", "CCC": "P", "CAC": "H", "CGC": "R",
+	"CUA": "L", "CCA": "P", "CAA": "Q", "CGA": "R",
+	"CUG": "L", "CCG": "P", "CAG": "Q", "CGG": "R",
+	"AUU": "I", "ACU": "T", "AAU": "N", "AGU": "S",
+	"AUC": "I", "ACC": "T", "AAC": "N", "AGC": "S",
+	"AUA": "I", "ACA": "T", "AAA": "K", "AGA": "R",
+	"AUG": "M", "ACG": "T", "AAG": "K", "AGG": "R",
+	"GUU": "V", "GCU": "A", "GAU": "D", "GGU": "G",
+	"GUC": "V", "GCC": "A", "GAC": "D", "GGC": "G",
+	"GUA": "V", "GCA": "A", "GAA": "E", "GGA": "G",
+	"GUG": "V", "GCG": "A", "GAG": "E", "GGG": "G",
+}
+
+// RNAToProtein accepts a RNA Sequence, translation table from NIBH standard and returns a protein sequence
+func RNAToProtein(rna []byte, transl_table int) (protein []byte, err error) {
```

Comment:
- Do not use named return variable `protein` here, because we can initialize it's `cap` according to size of `rna`.